src/Blog/BlogPlaceholder.scss

```diff
@@ -0,0 +1,123 @@
+@import "../styles/typography";
+.blog-section {
+  background-image: url("../../assets/blogbackground.jpg");
+  opacity: 0.75;
+  background-size: cover;
+  height: 100vh;
+  display: flex;
```

Comment:
This doesn't really need to be flexed. If you make the SVG 100% width it will center itself and the p element already stacks underneath and is full width anyway.

Reply:
Adding the flex so that I justify the content to sit in the middle of the page vertically?